main.go

```diff
@@ -2,26 +2,60 @@ package main
 
 import (
 	"log"
-	"os"
 
+	"github.com/alecthomas/kong"
 	"github.com/bobuhiro11/gokvm/flag"
 	"github.com/bobuhiro11/gokvm/probe"
 	"github.com/bobuhiro11/gokvm/vmm"
 )
 
-func main() {
-	// This line break is required by golangci-lint but
-	// such breaks are considered an anti-pattern
-	// at Google.
-	c, err := flag.ParseArgs(os.Args)
+type context struct {
+	Debug bool
```

Comment:
> flag.BoolVar(&c.Debug, "debug", false, "This flag provides a list of KVM capabilities and nothing more")

The debug option was previously used for this purpose, but this has been separated as a subcommand. So I think it is no longer needed.

Reply:
done